include/deal.II/lac/trilinos_vector.h

```diff
@@ -75,21 +211,29 @@ namespace TrilinosWrappers
 
   /**
    * Namespace for Trilinos vector classes that work in parallel over MPI.
-   * This namespace is restricted to vectors only, whereas matrices are always
-   * MPI based when run on more than one processor.
    *
    * @ingroup TrilinosWrappers
-   * @author Martin Kronbichler, Wolfgang Bangerth, 2008
+   * @author Martin Kronbichler, Wolfgang Bangerth, Daniel Arndt, 2008, 2017
    */
   namespace MPI
   {
     class BlockVector;
 
     /**
      * This class implements a wrapper to use the Trilinos distributed vector
-     * class Epetra_FEVector. This class is derived from the
-     * TrilinosWrappers::VectorBase class and provides all functionality
-     * included there.
+     * class Epetra_FEVector, the (parallel) partitioning of which
+     * is governed by an Epetra_Map.
+     * The Epetra_FEVector is precisely the kind of vector
+     * we deal with all the time - we probably get it from some assembly
+     * process, where also entries not locally owned might need to written and
+     * hence need to be forwarded to the owner. The only requirement for this
+     * class to work is that Trilinos is installed with the same compiler as is
+     * used for compilation of deal.II.
```

Comment:
We might as well fix this while we are here: I don't think that we need to get into a discussion of ABI compatibility in this class, so we should remove the last sentence.